include/pybind11/pytypes.h

```diff
@@ -2583,7 +2583,8 @@ str_attr_accessor object_api<D>::doc() const {
 
 template <typename D>
 object object_api<D>::annotations() const {
-#if PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION <= 9
+#if PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION <= 9                                                \
```

Comment:
I like the suppressed due to low confidence comment, but instead we should just use the HEX version like everywhere else.